dev-docs/howtoguides/spellcheck.md

```diff
@@ -0,0 +1,20 @@
+# How to spellcheck messages
+
+We have CI that runs on every PR which looks for spelling errors in our messages module, but it is often useful to run it locally as well.
```

Comment:
What language(s) does this check?

Reply:
Only english for now.

@s-makin is this better?
```suggestion
We have CI that runs on every PR which looks for spelling errors in our English messages, but it is often useful to run it locally as well.
```

Reply:
Fine by me - just so anyone working on translations doesn't waste their time running it :)